gateway-ha/src/test/java/io/trino/gateway/proxyserver/TestProxyRequestHandler.java

```diff
@@ -53,6 +53,7 @@ final class TestProxyRequestHandler
     private static final String OK = "OK";
     private static final int NOT_FOUND = 404;
     private static final MediaType MEDIA_TYPE = MediaType.parse("application/json; charset=utf-8");
+
```

Comment:
This change is unrelated to "Add support for request and response compression". Move to the 1st commit.